addon/components/inputs/select.js

```diff
@@ -48,6 +48,41 @@ export default AbstractInput.extend({
     return false
   },
 
+  @readOnly
+  @computed('bunsenModel', 'cellConfig.renderer.options')
+  listData (bunsenModel, renderOptions) {
+    const enumDef = bunsenModel.items ? bunsenModel.items.enum : bunsenModel.enum
+    const hasOverrides = !_.isEmpty(_.get(renderOptions, 'data'))
+    const hasNoneOption = _.get(renderOptions, 'none.present')
+
+    let data = []
+
+    if (enumDef && !hasOverrides) {
+      data = listUtils.getEnumValues(enumDef)
+    } else if (hasOverrides) {
+      data = _.cloneDeep(renderOptions.data)
+    }
+
+    if (hasNoneOption) {
+      const none = _.defaults({
+        label: renderOptions.none.label,
+        value: renderOptions.none.value
+      }, {
+        label: 'None',
+        value: ''
+      })
+      data = [none].concat(data)
+    }
+
+    return data
+  },
+
+  @readOnly
+  @computed('cellConfig.renderer.options.localFiltering')
```

Comment:
I've been trying to avoid using computed decorators on nested properties to make sure Ember doesn't convert our POJOs to Ember Objects. Instead I'd just do:

```js
@readOnly
@computed('cellConfig')
isFilteringLocally (cellConfig) {
  return _.get(cellConfig, 'renderer.options.localFiltering') || false
},
```